### DISCUSSION TITLE
Block commands (regex?)

### DISCUSSION THREAD

**Discussion by d4kir92** [night/betterttv#6194] [Ideas]
is it possible to hide or better delete messages with starting "!"

Examples:
!rip
!deaths
!youtube
!giveaway

i wanted to hide all commands not just some keywords*
the giveaways mostly have a new world each day on some streamers*

regex blocking would help or an extra checkbox to block commands

**Comment by d4kir92** [comment]
![image](https://github.com/night/betterttv/assets/23246564/420f7f2d-2d4b-44aa-a372-dea2106b8d47)

this is how FFZ does it*

**Comment by night** [comment]
There already is regex support. You can use the syntax `~/{REGEX}/{FLAGS}` to define one.

For example: `~/^!/`
